packages/office-ui-fabric-react/src/components/Divider/examples/VerticalDivider.Custom.Example.tsx

```diff
@@ -45,9 +46,11 @@ export class VerticalDividerCustomExample extends React.Component<any, any> {
     const exampleClassNames = getExampleClassNames();
     return (
       <div className={exampleClassNames.wrapper}>
-        <p className={exampleClassNames.text}> Some text before the divider. </p>
-        <VerticalDivider getClassNames={getVerticalDividerClassNames} />
-        <p className={exampleClassNames.text}>Some text after the divider. </p>
+        <Customizer settings={{ theme: { palette: { themePrimary: 'pink' } } }}>
+          <p className={exampleClassNames.text}> Some text before the divider. </p>
+          <VerticalDivider getClassNames={getVerticalDividerClassNames} />
```

Comment:
You might wanna remove the usage of the `getClassNames` prop and also, mark it as deprecated in the types now that you've introduced the `styles` prop.